tests/benchmark/fetch-download.js

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env node
+"use strict";
+
+import url from "node:url";
+import { Worker } from "node:worker_threads";
+
+const __dirname = url.fileURLToPath(new URL(".", import.meta.url));
+
+const TEST_RELEASE_BUILD = +process.env.TEST_RELEASE_BUILD;
+const USE_VIRTIO = !!process.env.USE_VIRTIO;
+const SERVER_PORT = parseInt(process.env.SERVER_PORT, 10) || 8686;
+const BENCHFILE_SIZE = (parseInt(process.env.BENCHFILE_SIZE_MB, 10) || 32) * 1024 * 1024;
+
+const { V86 } = await import(`../../build/${TEST_RELEASE_BUILD ? "libv86" : "libv86-debug"}.mjs`);
```

Comment:
```suggestion
const { V86 } = await import(`../../build/libv86.mjs`);
```

(I think debug mode should be off for benchmarks to reduce the noise)